docker/Dockerfile

```diff
@@ -0,0 +1,62 @@
+FROM ubuntu:20.04 as web-base
+
+RUN apt update && apt install -y git curl
+RUN curl -sL https://deb.nodesource.com/setup_14.x -o nodesource_setup.sh && \
+    bash ./nodesource_setup.sh && \
+    apt install nodejs && \
```

Comment:
This installs NodeJS v14. The SDK's `package-lock.json` is version 2 which implies NodeJS v16. There are some warning because of this. Might need to use NVM or similar to get a different NodeJS version.
Also, v-web-sdk should probably have an "engine:" section added to its `package.json`.